tests/test_transformer/test_transformer_examples.py

```diff
@@ -33,3 +43,18 @@ def test_examples(self):
             target_obj = tr.transform(input_obj)
             # target_obj = dynamic_object(target_obj, "Person")
             self.assertEqual(expected, target_obj["height_in_cm"])
+
+    def test_all(self):
+        """
+        Iterates through all examples
+        :return:
+        """
+        mft = MultiFileTransformer()
+        dirs = ["measurements"]
+        for dir in dirs:
+            full_dir = EXAMPLE_DIR / dir
+            instructions = mft.infer_instructions(full_dir)
+            print(instructions)
+            obj = mft.process_instructions(instructions, full_dir)
+            print(obj)
+
```

Comment:
it doesn't look like `mft.process_instructions` returns anything

Reply:
fixed thx!